sm2a/infrastructure/main.tf

```diff
@@ -91,6 +91,7 @@ module "sma-base" {
     VECTOR_SECRET_NAME    = var.vector_secret_name
     ASSUME_ROLE_READ_ARN = var.assume_role_read_arn
     ASSUME_ROLE_WRITE_ARN = var.assume_role_write_arn
+    SM2A_BASE_URL = module.sma-base.airflow_url
```

Comment:
Does the module have `airflow_url`? https://github.com/NASA-IMPACT/self-managed-apache-airflow/blob/main/infrastructure/varaibles.tf

I can't find it. I think the file I linked above should also be renamed to `variables.tf` (just a small typo).

Reply:
It does have it as an output: https://github.com/NASA-IMPACT/self-managed-apache-airflow/blob/main/infrastructure/outputs.tf#L5
We are using the same output to show SM2A URL when we deploy https://github.com/NASA-IMPACT/veda-data-airflow/blob/dev/sm2a/infrastructure/outputs.tf#L1